notify/options.go

```diff
@@ -24,7 +24,7 @@ import (
 )
 
 // Option is a functional option supplied to Init.
-type Option func(*Notifier) error
+type Option func(*MailJetNotifier) error
```

Comment:
NewMailJetNotifier -> NewMailjetNotifier

Perhaps Option should become MailjetOption. Although most options are generic and have nothing to do with Mailjet.